DCC.schema.json

```diff
@@ -3,32 +3,53 @@
   "$id": "https://id.uvci.eu/DCC.schema.json",
   "title": "EU DCC",
   "description": "EU Digital Covid Certificate",
-  "$comment": "Schema version 1.2.1",
-  "required": [
-    "ver",
-    "nam",
-    "dob"
-  ],
+  "$comment": "Schema version 1.3.0",
   "type": "object",
+  "oneOf": [
+    {
+      "required": [
+        "ver",
+        "nam",
+        "dob",
+        "v"
+      ]
+    },
+    {
+      "required": [
+        "ver",
+        "nam",
+        "dob",
+        "t"
+      ]
+    },
+    {
+      "required": [
+        "ver",
+        "nam",
+        "dob",
+        "r"
+      ]
+    }
+  ],
   "properties": {
     "ver": {
       "title": "Schema version",
       "description": "Version of the schema, according to Semantic versioning (ISO, https://semver.org/ version 2.0.0 or newer)",
       "type": "string",
       "pattern": "^\\d+.\\d+.\\d+$",
       "examples": [
-        "1.2.1"
+        "1.3.0"
       ]
     },
     "nam": {
-      "description": "Surname(s), given name(s) - in that order",
+      "description": "Surname(s), forename(s) - in that order",
       "$ref": "https://id.uvci.eu/DCC.Core.Types.schema.json#/$defs/person_name"
     },
     "dob": {
       "title": "Date of birth",
       "description": "Date of Birth of the person addressed in the DCC. ISO 8601 date format restricted to range 1900-2099",
       "type": "string",
-      "pattern": "^(19|20)\\d\\d(-\\d\\d){0,2}$",
+      "pattern": "^((19|20)\\d\\d(-\\d\\d){0,2}){0,1}$",
       "examples": [
         "1979-04-14",
         "1950",
```

Comment:
Add YYYY-MM example

Reply:
Can do - usually the examples aren't an exhaustive list of all possibilities, but if it makes you feel better, then sure :)